test/Tests/Strings/LoopyAutomatonTests.cs

```diff
@@ -792,8 +792,7 @@ private static double GetLogNormalizerByGetValueWithTransducers(StringAutomaton
         /// <param name="automaton">The automaton.</param>
         private static void AssertStochastic(StringAutomaton automaton)
         {
-            StringAutomaton automatonClone = automaton.Clone();
-            automatonClone.RemoveDeadStates();
+            automaton.RemoveDeadStates(out var automatonClone);
```

Comment:
This looks awkward.  It should be `automatonClone = automaton.RemoveDeadStates()`.  The boolean return value does not seem important.